gems/smithy-client/lib/smithy-client/stubbing/data_applicator.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Client
+    module Stubbing
```

Comment:
Should we have a file, `stubbing.rb?` so we can mark that module api-private? Showing up on rendered docs.